.flake8

```diff
@@ -1,4 +1,3 @@
 [flake8]
 max-line-length = 110
 ignore = E731
-exclude = .git, env, venv, .venv, VENV, .tox, .eggs, .*
```

Comment:
Is the problem from the ".*" or does any exclude cause this problem?

Reply:
I'm not completely sure, but I believe it's the `.*`.

But before we trouble-shoot thoroughly, I would suggest to revert the whole line to ensure that Travis CI works as expected.

Reply:
Makes sense :)